rbs-inline.gemspec

```diff
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "prism", ">= 0.29", "< 1.3"
+  spec.add_dependency "prism", ">= 0.29", "< 1.5"
```

Comment:
It doesn't seem necessary to lock the upper (minor) version bound of Prism. I've heard from user feedback that they were unable to resolve versions when using the latest RuboCop (RuboCop AST), which requires Prism 1.4+. For reference, RuboCop has no plans to impose an upper bound on minor versions of Prism:
https://github.com/rubocop/rubocop-ast/blob/v1.44.0/rubocop-ast.gemspec#L37

It's probably better to leave the Prism version up to users, I think.